### PR TITLE
New Song for Map 20 for Freedoom phase 2

### PR DESCRIPTION
**Pull request by Josephus-DH4050-Astartes**
the replacement song (Beasts Of The Horizon - Josephus "DH4050" Astartes)

### CODE REVIEW

**Comment by unithecutie** [comment]
Update CREDITS and CREDITS-MUSIC as well in the PR please!
Also for "Previously Featured" for E2M8:

E2M8: "Pit" by Anthony "Ajanddino" Pierce
image

**Review by mc776 — APPROVED** [review]
I just gave this and the current track a quick try, playing up to the end of the quadruped room one right after the other.

Am actually, positively *enjoying myself* more on this map with this than the current track.

I'll leave this open until tomorrow just in case anyone's got any other comments but right now I don't see any reason not to merge this.

File: CREDITS-MUSIC [hunk 1]
```diff
@@ -80,7 +80,7 @@ Cluster 2
 	MAP17: "Mow Them Down!" by Lola "BlueWorrior" Harvey
 	MAP18: "Realistic Approach" by Kevin "Velvetic" Martins
 	MAP19: "Deceit" by Julian "Julian Hope" Aubourg, modified by m
-	MAP20: dave3d05.mid by jute
+	MAP20: "Beasts of THe Horizon" by Josephus "DH4050" Astartes
```
Comment (mc776) [inline CREDITS-MUSIC:83]: "THe"?

**Review by mc776 — CHANGES_REQUESTED** [review]

File: CREDITS-MUSIC (hunk 1 above)
Comment (Josephus-DH4050-Astartes) [reply CREDITS-MUSIC:83]: oops XD anyways, im glad you enjoyed it! ^^

Comment (Josephus-DH4050-Astartes) [reply CREDITS-MUSIC]: alright, i've fixed it. ^^

**Review by mc776 — APPROVED** [review]

**Pull request merged**